Frontend-Angular/data-polish-frontend/src/assets/Dataprofile_config.json

```diff
@@ -11,29 +11,89 @@
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "NUMBEROFRECORDS":{
+        "WHAT":"Displays the count of Records in the dataset.",
+        "WHY":"In order to understand the Dataset size"
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "bubble-chart":{
+        "WHAT":"Visualises the categorical features within a column as bubbles.",
+        "WHY":"Use to identify categorical distribution. Large bubbles may indicate asimetric data points needing normalisation."
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "cardinality":{
+        "WHAT":"Shows the uniqueness of data by representing the number of distinct values in a column.",
+        "WHY":"Helpful for understanding when to collapse rare categories or when binning might be necessary for numeric to categorical conversions."
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "correlation-scatter-plot":{
+        "WHAT":"Illustrates the relationship between two variables and their correlation coefficient.",
+        "WHY":"Use to determine if fields are redundant (high correlation) or if PCA might be useful for dimensionality reduction."
```

Comment:
Would remove the line about PCA possibly as we dont have that in the app anymore

Reply:
Or keep it, not sure will let you decide, users will have to apply PCA out of the app